### <center>**Analyse des Clusters**</center>

#### **1. Calcule des Moyennes par Cluster :**

In [10]:
import pandas as pd

data = pd.read_csv("../data/trained/data.csv")

data.groupby('Cluster').mean()

Pregnancies   Glucose  BloodPressure  SkinThickness   Insulin  \
Cluster                                                                  
0           1.509682  1.040599       4.333613       3.460705  5.314073   
1           1.191172 -0.624359       4.249242       3.274061  4.556771   

              BMI  DiabetesPedigreeFunction       Age  
Cluster                                                
0        3.556648                  0.390219  3.614360  
1        3.449156                  0.350377  3.401902

- **Cluster 0 - Potentiel risque élevé :**

    - ``Glucose`` : Moyenne élevée (1.04), au-dessus de la moyenne globale càd risque plus élevé de diabète.

    - ``Insulin`` : Moyenne légèrement élevée (5.31) ce qui reflète un métabolisme actif de l’insuline.

    - ``BMI`` : Légèrement au-dessus de la moyenne (3.56) càd un poids légèrement supérieur.

    - ``Pregnancies``, ``BloodPressure``, ``SkinThickness``, ``Age``, ``DiabetesPedigreeFunction`` : Légèrement supérieurs à la moyenne globale, contribuant modérément au profil de risque.

    - **Interprétation générale :** Ce cluster regroupe les individus présentant un profil métabolique plus élevé, donc un risque potentiel de diabète plus important.

- **Cluster 1 - Potentiel risque faible :**

    - ``Glucose`` : Moyenne inférieure (-0.62), au-dessous de la moyenne globale càd risque de diabète plus faible.

    - ``Insulin`` : Moyenne plus basse (4.56) càd activité ``insulin``ique moins élevée.

    - ``BMI`` : Légèrement plus faible (3.45) càd poids inférieur à la moyenne globale.

    - ``Pregnancies``, ``BloodPressure``, ``SkinThickness``, ``Age``, ``DiabetesPedigreeFunction`` : Légèrement inférieurs ou proches de la moyenne globale, contribuant faiblement au risque.

    - **Interprétation générale :** Ce cluster regroupe les individus présentant un profil métabolique moins prononcé, donc un risque potentiel de diabète plus faible.